### DISCUSSION TITLE
Do not hijack the browser's find menu

### DISCUSSION THREAD

**Discussion by skington** [community/community#54434] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In some screens (e.g. blame), if I hit command-F, I get a custom find dialogue box instead of my browser's find menu.
<img width="387" alt="Screenshot 2023-05-04 at 12 13 48" src="https://user-images.githubusercontent.com/845347/236188388-9f59cdb8-0e46-4c87-9924-58a90902ae49.png">

This is uncalled for and I want a way to turn it off, the way I can turn off other uncalled-for keyboard shortcuts, for two reasons.

First of all: I didn't ask for this, and I don't see why it's any better than my browser's find menu.

Secondly: you don't hijack command-G, so so when it hit command-G to find the next instance of what I was searching for, I almost certainly search for *something else*.

**Comment by spenserblack** [comment]
I think that, ideally, GitHub could add an accessibility option to change the behavior of <kbd>ctrl</kbd>+<kbd>f</kbd>. Maybe a toggle between <kbd>ctrl</kbd>+<kbd>f</kbd> and <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>f</kbd>, like they did for URL paste behavior?

**Comment by spazmodius** [comment]
~So aggravating~

I realize now that's because of some feature preview, I can just turn off.

**Reply by anne-gropler** [reply]
Where can I turn this off?

**Comment by julianoes** [comment]
It looks like good old Ctrl+F like the browser is perfectly capable of doing is no longer a thing. Can this be disabled?
